test/unit/adapters/sequelize.test.js

```diff
@@ -1743,11 +1741,22 @@ describe('validator should work', () => {
         status: 1,
       }, { validate: false });
 
+      await new Promise((resolve) => {
+        setTimeout(() => {
+          resolve(true);
+        }, 1000);
+      });
```

Comment:
why is this sleep necessary?

Reply:
Verifying createdAt will not be updated at `upsert`